types/rsmq-worker/index.d.ts

```diff
@@ -18,6 +18,7 @@ declare namespace RSMQWorker {
         send(message: string, cb: CallbackT<string>): Client;
         del(id: string, cb?: CallbackT<void>): Client;
         changeInterval(interval: number | number[]): Client;
+        quit(): void;
```

Comment:
It looks like this method was added on version 0.4.2. Could you update the header above to match that version? Thanks.